index.html

```diff
@@ -643,6 +643,16 @@ <h2>Privacy and Security</h2>
          
      </section>
     
+     <section class='informative'>
+      <h2>Privacy Considerations</h2>
+      <p>
+          MiniApp Global Application Lifecycle interface {{Global}} introduces {{inputObject}} involves the processing of the inputted query for the MiniApp, the page path for current MiniApp, and the source info. MiniApp Page Lifecycle interface {{Page}} introduces {{pageInputObject}} involves the processing of the inputted query for the MiniApp page. In order to protect users from any potential unsanctioned tracking threat, it is not recommended to store these values locally. If these values are stored, these storages should be cleared when users intend to clear them.
+      </p>
+      <p>
+         If the inputted query for the MiniApp or the inputted query for the MiniApp page contains privacy-sensitive information (e.g. user personal data), the privacy-sensitive information shall not be in plaintext format.
```

Comment:
```suggestion
         If the inputted query for the MiniApp or the inputted query for the MiniApp page contains privacy-sensitive information (e.g. user personal data), the privacy-sensitive information shall not be in cleartext.
```

[Cleartext](https://www.hypr.com/cleartext/) should be better here.